frontend/sw360-portlet/src/main/webapp/html/projects/includes/linkedProjects.jspf

```diff
@@ -44,7 +46,7 @@
         >
             <td>
                 <a href="<sw360:DisplayProjectLink projectId="${projectLink.id}" bare="true" />"><sw360:out
-                        value="${projectLink.name}"/></a>
+                        value="${projectLink.name}" maxChar="50"/></a>
```

Comment:
No, I didn't. I went with what was at hand, I admit. I'll try and test this option